utils/OffchainStepper.js

```diff
@@ -129,6 +143,7 @@ class RangeProofHelper {
     this.readHigh = -1;
     this.writeLow = -1;
     this.writeHigh = -1;
+    this.readAndWrites = [];
```

Comment:
I'm not sure I understand RangeProofHelper from last time, but with this new one, isn't we don't need `readHigh`, `readLow`, `writeHigh`, and `writeLow` anymore?

Reply:
🤔 Yes, we could probably clean those up if we going to implement using the `FragmentTree` for calldata and memory 🙂

Reply:
The generic idea is to just have lowest accessed byte and highest accessed byte, because calldata and memory usually spans just across one word. And in cases like `staticcall` to precompiles a bigger range close together.

Reply:
I see. Let's decide that on the issue about memory and calldata.